doc/api/cli.md

```diff
@@ -288,6 +288,17 @@ Path to the file used to store the persistent REPL history. The default path is
 to an empty string (`""` or `" "`) disables persistent REPL history.
 
 
+### `NODE_TTY_UNSAFE_ASYNC=1`
+<!-- YAML
+added: REPLACEME
+-->
+
+When set to `1` writes stdout and stderr will be non-blocking and async when
+outputting to a TTY on platforms which support async stdio.
+Setting this will void any guarantee that stdio will not be interleaved or
+dropped at program exit. **Avoid use.**
```

Comment:
perhaps soften, something like `**Use of this mode is not recommended.** The name "UNSAFE" already is already blunt enough I reckon.